src/components/login/LoginForm.js

```diff
@@ -12,7 +12,7 @@ import style from '../../style/style';
 /** Various props that control how the user interacts with
  *  the login form
  * @type {React.FunctionComponent}
- * @param {object} props
+ * @param {ILoginFormComponentProps} props
  * @param {any} props.error
  * @param {object} props.classes
```

Comment:
You don't need these items now that ILoginFormComponentProps includes them

Reply:
Fixed